numpy/__init__.pyi

```diff
@@ -899,6 +902,11 @@ _ArrayLikeIntOrBool = Union[
     Sequence[_ArrayLikeIntNested],
     Sequence[_ArrayLikeBoolNested],
 ]
+_ArrayLikeBool = Union[
+    _Bool,
+    Sequence[_Bool],
+    ndarray
```

Comment:
Soon we will have ndarray generic over dtype...

Reply:
Yeah, I feel this is gona be a *biiiiiig* help.